
# Welcome <a id="welcome"></a>

Welcome to MICB405 Project 1: Epigenomic Data Analysis.  

In this hands on project you will work in teams to analyze the epigenetic state changes in the genomes of murine CD4 T-cells associated with influenza challenge.  We will focus our analysis on a histone modification that is associated with [active regulatory states] (https://github.com/martinhirst/micb405_project_1/blob/master/workflow.jpeg) in mammalian genomes. 

The workflow for this project can be found [here](https://github.com/martinhirst/micb405_project_1/blob/master/workflow.jpeg).

1)	Completion at different stages of the workflow (output file in correct formats)
2)	Structure of the written report: Abstract (250 words), Introduction (500-1000 words), Methods (1000-2000 words including workflow, directory structure, commands with parameter selection rationale in written report), Results (1000-2000), unlimited figures, tables and references.
3)	Structure of the presentation (follows general structure of the written report with emphasis on results)


Pre-readings, pre-work, and laptop setup instructions can be found [here](https://bioinformaticsdotca.github.io//Epigenomics_2017_prework)

If you have questions or comments, visit the workshop [Q&A forum](https://noteapp.com/Epi2017).

***

# Day 1 <a id="day1"></a>

##  Module 1: Introduction to ChIP Sequencing and Analysis <a id="module_1"></a>

  *<font color="#827e9c">Martin Hirst</font>*
  
  [Lecture](https://bioinformatics.ca/epigenomics-module-1-2016)
  
  [Lecture on Compute Canada](https://bioinformatics.ca/epigenomics-module-1-hpc-2016)
  
  [Using the Compute Canada HPC](https://bioinformaticsdotca.github.io/epigenomics_2017_hpc_2017)

  [Lab practical](http://bioinformatics-ca.github.io/epigenomic_data_analysis_module1_lab_2016/)
  
  
##  Module 2: ChIP-Seq Alignment, Peak Calling, and Visualization <a id="module_2"></a>

  *<font color="#827e9c">Misha Bilenky</font>*
  
  [Lecture](https://bioinformatics.ca/epigenomics-module-2-2016)
  
  [Lab practical](http://bioinformatics-ca.github.io/epigenomic_data_analysis_module2_lab_2016/)

## Integrated Assignment


# Day 2 <a id="day2"></a>

##  Module 3: Introduction to WGBS and Analysis <a id="module_3"></a>

  *<font color="#827e9c">Guillaume Bourque</font>*
  
  [Lecture](https://bioinformatics.ca/epigenomics-module-3-2016)
  
  [Lab practical](http://bioinformatics-ca.github.io/epigenomic_data_analysis_module3_lab_2016/)


##  Module 4: Downstream Analyses and Integrative Tools <a id="module_4"></a>

  *<font color="#827e9c">David Bujold</font>*
  
  [Lecture](https://bioinformatics.ca/epigenomics-module-4-2016)
  
  [Lab practical](http://bioinformatics-ca.github.io/epigenomic_data_analysis_module4_lab_2016/)

***
